app/models/manageiq/providers/ansible_tower/automation_manager/credential_mixin.rb

```diff
@@ -0,0 +1,134 @@
+module ManageIQ::Providers::AnsibleTower::AutomationManager::CredentialMixin
+  extend ActiveSupport::Concern
+
+  FIELDS = {
```

Comment:
I don't think this belongs in the mixin.  Not all of these fields apply to all credential types.